spec/text-editor-spec.coffee

```diff
@@ -2343,6 +2343,61 @@ describe "TextEditor", ->
           editor.backspace()
           expect(editor.lineTextForBufferRow(0)).toBe 'var  =  () {'
 
+    describe ".deleteToPreviousWordBoundary()", ->
+      describe "when no text is selected", ->
+        it "deletes to the previous word boundary", ->
+          editor.setCursorBufferPosition([0, 16])
+          editor.addCursorAtBufferPosition([1, 21])
+          [cursor1, cursor2] = editor.getCursors()
+
+          editor.deleteToPreviousWordBoundary()
+          expect(buffer.lineForRow(1)).toBe 'var quicksort =function () {'
+          expect(buffer.lineForRow(2)).toBe '  var sort = (items) {'
+          expect(cursor1.getBufferPosition()).toEqual [0, 15]
+          expect(cursor2.getBufferPosition()).toEqual [1, 13]
+
+          editor.deleteToPreviousWordBoundary()
+          expect(buffer.lineForRow(1)).toBe 'var quicksort function () {'
+          expect(buffer.lineForRow(2)).toBe '  var sort =(items) {'
+          expect(cursor1.getBufferPosition()).toEqual [0, 14]
+          expect(cursor2.getBufferPosition()).toEqual [1, 12]
+
+      describe "when text is selected", ->
+        it "deletes only selected text", ->
+          editor.setSelectedBufferRange([[1, 24], [1, 27]])
+          editor.deleteToEndOfWord()
```

Comment:
@lee-dohm Oops.. yes, probably. I don't have the test text handy so I can't verify, but it doesn't make any sense that the test doesn't even call the new function.